tests/test_metrics_comparative.py

```diff
@@ -32,6 +32,7 @@ def test_binary_jaccard_reference(truth_binary_values, prediction_binary_values)
     reference_value = sklearn.metrics.jaccard_score(
         np.expand_dims(truth_binary_values, axis=0),
         np.expand_dims(prediction_binary_values, axis=0),
+        average='micro',
```

Comment:
Since this is a binary classification, we shouldn't need to think about the averaging strategy. For what it's worth, you can read up on micro vs macro here: https://datascience.stackexchange.com/a/16001 . Note, most of our "aggregrate" metrics use a macro averaging strategy, except for the balanced multiclass accuracy, which uses micro averaging (the *micro* is what makes it *balanced*).

In this case, does the need to use `micro` go away if we get rid of the extra array wrapping?